### PR TITLE
First attempt to add package URLs to README.md files

### PR DESCRIPTION
**Pull request by hex3262**
This commit only treats flat packages. Nested packages such as Alpine could not be handled since the syntax of such README.md files is not coherent.

### CODE REVIEW

File: analysed-packages/OpenSSL/version-1.1.1q/README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 https://github.com/openssl/openssl/archive/refs/tags/OpenSSL_1_1_1q.tar.gz
 
+## Package URL (purl)
+
+pkg:none/github.com/openssl/OpenSSL@1.1.1q
```
Comment (pombredanne) [inline analysed-packages/OpenSSL/version-1.1.1q/README.md:7]: `none` is not a known package type.
`pkg:github/openssl/openssl@1.1.1q` would be correct here. (note that the repo name is lowercase)

File: analysed-packages/bzip2/version-1.0.8/README.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 https://gitlab.com/bzip2/bzip2/-/archive/bzip2-1.0.8/bzip2-bzip2-1.0.8.tar.gz
 
+## Package URL (purl)
+
+pkg:none/gitlab.com/bzip2/bzip2@1.0.8
```
Comment (pombredanne) [inline analysed-packages/bzip2/version-1.0.8/README.md:7]:
```suggestion
pkg:gitlab/bzip2/bzip2@1.0.8
```

File: analysed-packages/zlib/version-1.2.12/README.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 https://zlib.net/fossils/zlib-1.2.12.tar.gz
 
+## Package URL (purl)
+
+pkg:none/zlib.net/fossils/zlib@1.2.12
```
Comment (pombredanne) [inline analysed-packages/zlib/version-1.2.12/README.md:7]:
```suggestion
pkg:generic/zlib@1.2.12?download_url=https://zlib.net/fossils/zlib-1.2.12.tar.gz
```

Comment (hex3262) [reply analysed-packages/zlib/version-1.2.12/README.md]: Thank you, considered your suggestions and adapted all other occurrences accordingly. Please see second attempt.

**Pull request closed without merging**